src/components/FavoritesPage.jsx

```diff
@@ -1,10 +1,19 @@
-function FavoritesPage() {
+import { CardContent, CardListStyled, CharacterName } from './CardList';
+import { Link } from 'react-router-dom';
 
+
+function FavoritesPage({ favoriteCharacters }) {
   return (
-    <>
-      <p>Favorites</p>
-    </>
+    <CardListStyled>
+      {favoriteCharacters.map(({ name, image, id}) => (
```

Comment:
Its better to filter the ids from localstorage with the original characters and display the filtered characters here